examples/js/loaders/GLTFLoader.js

```diff
@@ -2262,6 +2262,13 @@ THREE.GLTFLoader = ( function () {
 
 			if ( materialDef.name !== undefined ) material.name = materialDef.name;
 
+			// https://github.com/mrdoob/three.js/issues/11438#issuecomment-507003995
+			if ( material.normalScale && ! material.vertexTangents ) {
+
+				material.normalScale.y = - material.normalScale.y;
+
+			}
```

Comment:
Can you check that this is OK on a model with `vertexTangents`? Note that we're in the `loadMaterial` method here, and `.vertexTangents` wouldn't be set until later, in the `assignFinalMaterial` method, when we have access to the mesh and can confirm whether it has tangents. I think the change will need to be moved there.

Reply:
Indeed, found the repro with your shoe.glb. Moved and verified the fix. Thanks!